### ISSUE TITLE
The Code used for the training stops

### ISSUE THREAD

**Issue by anirban1513** [Project-MONAI/tutorials#71]
!python run_net.py train --data_folder "COVID-19-20_v2/Train" --model_folder "runs"

Code cell stops after showing this result. Don't know what should I do next, How can I find the models? 

MONAI version: 0.3.0+87.ge94e243
Python version: 3.6.9 (default, Oct  8 2020, 12:12:24)  [GCC 8.4.0]
OS version: Linux (4.19.112+)
Numpy version: 1.18.5
Pytorch version: 1.7.0+cu101
MONAI flags: HAS_EXT = False, USE_COMPILED = False

Optional dependencies:
Pytorch Ignite version: 0.4.2
Nibabel version: 3.0.2
scikit-image version: 0.16.2
Pillow version: 7.0.0
Tensorboard version: 2.3.0
gdown version: 3.6.4
TorchVision version: 0.8.1+cu101
ITK version: NOT INSTALLED or UNKNOWN VERSION.
tqdm version: 4.53.0

For details about installing the optional dependencies, please visit:
    https://docs.monai.io/en/latest/installation.html#installing-the-recommended-dependencies

INFO:root:training: image/label (199) folder: COVID-19-20_v2/Train
INFO:root:training: train 160 val 39, folder: COVID-19-20_v2/Train
INFO:root:batch size 2
Load and cache transformed data: 100% 160/160 [05:30<00:00,  2.06s/it]
Load and cache transformed data: 100% 39/39 [01:24<00:00,  2.18s/it]
BasicUNet features: (32, 32, 64, 128, 256, 32).
^C

**Comment by wyli** [comment]
see https://github.com/Project-MONAI/tutorials/issues/67
